### PR TITLE
fix: update Carbon dependencies & update snapshots

### LINKED ISSUE #209: [Accordion] add `align` prop to skeleton component
Context: https://github.com/carbon-design-system/ibm-security/pull/205#pullrequestreview-313274742

Currently the `Accordion` has an `align` prop for start/end alignment, but the `AccordionSkeleton` does not. The skeleton component should have the same prop so that it could match.

### LINKED ISSUE #281: Update to latest Carbon release v10.9.0
Update to latest Carbon release: https://github.com/carbon-design-system/carbon/releases/tag/v10.9.0

### PR DESCRIPTION
**Pull request by jendowns**
## Affected issues

- Resolves #281 
- Resolves #209 

## Proposed changes

- update Carbon dependencies (NOTE: *must* use alpha version of `carbon-components` due to https://github.com/carbon-design-system/carbon/issues/4926)
- add style rule to dropdown button in `ComboButton`, since Carbon added a reset to their core dropdown.
- update snapshots
- **Specific `Accordion` updates --**
  - remove unnecessary start/end classes in security namespace
  - remove unnecessary style overrides for `start` alignment
  - update stories to match Carbon's more closely
  - add `align` prop to `AccordionSkeleton` story specifically
  - add a style override for misaligned icon if `AccordionSkeleton` alignment is set to `end`

## Testing instructions

- Check out the storybook to verify the components look good to go

### CODE REVIEW

**Comment by jendowns** [comment]
@SimonFinney In my most recent commit I also made some changes to the `Accordion` --

Since Carbon now offers an `align` prop, there's no need for our alignment classes + style overrides, so I removed those. I also updated the `Accordion` & `AccordionSkeleton` stories to match Carbon's more closely.

File: docs/themes/themes.md [hunk 1]
```diff
@@ -96,3 +96,33 @@
 | `display-02`              | `colors`          | `colors`          |
 | `display-03`              | `colors`          | `colors`          |
 | `display-04`              | `colors`          | `colors`          |
+| `spacing-01`              | `colors`          | `colors`          |
```
Comment (jendowns) [inline docs/themes/themes.md]: NOTE: this is automatic and will go away after #289 is merged

File: src/components/Accordion/Accordion.js [hunk 2]
```diff
@@ -3,39 +3,25 @@
...
 const { defaultProps, propTypes } = CarbonAccordion;
 
 const Accordion = ({ className, align, ...other }) => (
-  <CarbonAccordion
-    className={classnames(namespace, className, {
-      [`${namespace}--${align}`]: align,
-    })}
-    {...other}
-  />
+  <CarbonAccordion {...other} align={align} />
```
Comment (SimonFinney) [inline src/components/Accordion/Accordion.js]: 👏 Can we remove this component entirely and just refer to Carbon's?

File: src/components/ComboButton/_index.scss [hunk 3]
```diff
@@ -4,6 +4,9 @@
 /// @copyright IBM Security 2019
 ////
 
+@import 'carbon-components/scss/components/button/button';
```
Comment (SimonFinney) [inline src/components/ComboButton/_index.scss]: What are your thoughts on importing our `Button` and `Dropdown` styles here versus Carbon's?

File: src/globals/type/_index.scss [hunk 4]
```diff
@@ -15,6 +15,22 @@
 @import '../feature-flags/index';
 @import '../namespace/index';
 
+// TODO: Remove when fix is released in `@carbon/type` (https://github.com/carbon-design-system/carbon/pull/4927)
+
+$expressive-heading-01: map-merge(
```
Comment (SimonFinney) [inline src/globals/type/_index.scss]: 👏

File: src/components/ComboButton/_index.scss (hunk 3 above)
Comment (jendowns) [reply src/components/ComboButton/_index.scss]: Sure I can do that

File: src/components/Accordion/Accordion.js (hunk 2 above)
Comment (jendowns) [reply src/components/Accordion/Accordion.js]: No we still need `align="start"` by default, the opposite of Carbon's default.

Comment (SimonFinney) [reply src/components/Accordion/Accordion.js]: Oh! Good point!

**Comment by jendowns** [comment]
Thanks @SimonFinney I made the import update you suggested above 👍

**Pull request merged**